src/farmers/farmerbot/farmerbot_quick.md

```diff
@@ -40,76 +68,184 @@ Here are some examples to guide you:
 
-* Deploy a full VM on the TFGrid with minimum specs (512 MB of memory, 1 vcore, 15 GB of storage)
-  * Read [this guide](../../getstarted/ssh_guide/ssh_guide.md) for more information on deploying a Full VM with the TF Playground
-
-***
+- [Deploy full vm on the Playground with WireGuard network](../../getstarted/ssh_guide/ssh_guide.md)
+- Update and upgrade the VM
+    ```
+    apt update && apt upgrade
+    ```
+- Reboot and reconnect to the VM
+    ```
+    reboot
+    ```
 
-## Farmerbot Costs on the TFGrid
+## Method 1: Binaries and Zinit
 
-If you run the Farmerbot on a 3Node on the TFGrid, you will have to pay TFT to deploy on that 3Node. You can run a full VM at minimum specs for the Farmerbot, that is 1vcore, 15GB of SSD storage and 512MB of RAM. Note that you can use the Planetary Network. You do not need to deploy a 3Node with IPv4. The cost on main net for this kind of workload is around 0.175TFT/hour (as of the date 11-07-23).
+We present how to run the Farmerbot using the binaries.
 
-Next to that, you will have to pay the transaction fees every time the Farmerbot has to wake up or shut down a node. This means that you need some TFT on the account tied to the twin of your farm. 
+### Download the Farmerbot Binaries
 
-For the periodic wakeups, each node in the farm is shut down and powered on once a day, i.e. 30 times per month. Also, there is 10 random wakeups per month for each node. This means that each node is turned off and on 40 times per month in average. In that case, the average cost per month to power on nodes and shut them back down equals:
+- Download and Extract the [ThreeFold tfgrid-sdk-go release](https://github.com/threefoldtech/tfgrid-sdk-go/releases) for your specific setup (here we use `x86_64`)
+    ```
+    mkdir tfgrid-sdk-go
+    cd tfgrid-sdk-go
+    wget https://github.com/threefoldtech/tfgrid-sdk-go/releases/download/v0.13.16/tfgrid-sdk-go_Linux_x86_64.tar.gz
+    tar -xvf tfgrid-sdk-go_Linux_x86_64.tar.gz
+    ```
+- Move the Farmerbot
+    ```
+    mv farmerbot /usr/local/bin
+    ```
 
-> average transaction fees cost per month = 0.001 TFT (extrinsic fee) * amount of nodes * 40 * 2 (1 for powering down, one for powering up)
+### Create the Farmerbot Files
 
-***
+- Create Farmerbot files directory
+    ```
+    cd ~
+    mkdir farmerbotfiles
+    ```
+- Create the Farmerbot `config.yml` file (see template below)
+    ```
+    nano ~/farmerbotfiles/config.yml
+    ```
+- Create the environment variables file and set the variables (see template below)
+    ```
+    nano ~/farmerbotfiles/.env
+    ```
 
-## Install Docker
+### Run the Farmerbot
 
-For this guide, we need to install Docker Engine. The quickest way is to use the Docker convenience script. This method installs the latest stable release of Docker on Linux:
+We run the Farmerbot with the following command:
 
 ```
-curl -fsSL https://get.docker.com | sh
+farmerbot run -e ~/farmerbotfiles/.env -c ~/farmerbotfiles/config.yml -d
```

Comment:
Add a special note please for the key type `-k` parameter that allows farmers with ed25519 keys to use the farmerbot as well

Reply:
Done. I also added a quick url link to the additional information section, which covers all those nuances.